plugins/linear-genome-view/src/LinearGenomeView/components/ImportForm.tsx

```diff
@@ -62,6 +65,9 @@ const ImportForm = observer(({ model }: { model: LGV }) => {
     const newRegion = assemblyRegions.find(r => selectedRegion === r.refName)
     if (newRegion) {
       model.setDisplayedRegions([newRegion])
+      // we use showAllRegions after setDisplayedRegions to make the entire
+      // region visible, xref #1703
+      model.showAllRegions()
```

Comment:
do we have anything to keep from crashing their browser if there are way way too many regions in the assembly?